### PR TITLE
As a learner I can see all my questions in My Questions page

### PR DESCRIPTION
**Pull request by Amber-Williams**
Changed routes to work with params to show question components

### CODE REVIEW

File: src/components/question-posted-page/question-about.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      method: 'GET', 
+      headers : { 
+        'Authorization' : 'Bearer ' + token,
+        'Content-Type': 'application/json'
+      }})
+    .then(res => res.json())
+    .then(res => 
+      this.setState({
+        question: res.filter(ele => {
+          if (ele.questionId == window.location.pathname.replace(/\D/g, "")) {
```
Comment (nataliaero) [inline src/components/question-posted-page/question-about.js]: I would add === instead of ==

Comment (Amber-Williams) [reply src/components/question-posted-page/question-about.js]: @nataliaero the types are different but i'll add ele.questionId.toString()

Comment (Amber-Williams) [reply src/components/question-posted-page/question-about.js]: @nataliaero done!

**Review by nataliaero — APPROVED** [review]
Great!

**Pull request merged**